### PR TITLE
Go feedback

### LINKED ISSUE #118: connect to feedBack page

### PR DESCRIPTION
**Pull request by amanshawar**
connect the feedBack page from item list page  
fix #118

### CODE REVIEW

File: client/src/components/ItemList/index.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import React, { Component } from 'react';
 import axios from 'axios';
 import Title from '../Shared/Title';
 import GButton from '../Shared/GreenButton';
+import Button from '../Shared/Button';
+
```
Comment (jema28) [inline client/src/components/ItemList/index.js]: Why there's an extra line here?

File: client/src/components/ItemList/index.js [hunk 2]
```diff
@@ -31,6 +33,12 @@ class ItemList extends Component {
     history.push('/upload-photo');
   };
 
+  feedback = () => {
+    console.log('here');
```
Comment (jema28) [inline client/src/components/ItemList/index.js]: Please delete console logs <3

File: client/src/components/ItemList/index.js [hunk 3]
```diff
@@ -60,7 +68,8 @@ class ItemList extends Component {
         </List>
         <StyledBottom>
           <GButtonContainer>
-            <GButton title="EXPORT AS CSV" />
+            <Button title="EXPORT AS CSV" />
+            <GButton onClick={this.feedback} title="Give your feedback" />
```
Comment (jema28) [inline client/src/components/ItemList/index.js]: This needs to be in capitals to keep consistent with the other buttons.

**Review by jema28 — CHANGES_REQUESTED** [review]

File: client/src/components/Feedback/index.js [hunk 4]
```diff
@@ -150,7 +150,7 @@ class Feedback extends Component {
             </ThanksText>
           )}
           <Button />
-          <GButton title="CONTINUE" onClick={this.updateAirtaleFeedback} />
+          <GButton title="SAVE" onClick={this.updateAirtaleFeedback} />
```
Comment (orjwan98) [inline client/src/components/Feedback/index.js]: You have a typo here. You may want to check it out

**Review by jema28 — APPROVED** [review]

**Review by FidaaHersh — APPROVED** [review]

**Review by orjwan98 — APPROVED** [review]

**Review by jema28 — APPROVED** [review]

**Pull request merged**